CMakeLists.txt

```diff
@@ -88,7 +88,7 @@ ENDIF (ENABLE_PYTHON)
 ADD_SUBDIRECTORY (librepo)
 
 IF (ENABLE_TESTS)
-  FIND_LIBRARY(CHECK_LIBRARY NAMES check REQUIRED)
+  PKG_CHECK_MODULES(LIBCHECK REQUIRED IMPORTED_TARGET check)
```

Comment:
Is it necessary to use the `IMPORTED_TARGET` feature? I see it was added in CMake 3.6, the file claims minimum required version 2.8.5 and we should stick with that, maintaining compatibility with RHEL 7.

Reply:
That's what makes the `PkgConfig::LIBCHECK` below working. I'll try to switch to some manual glueing to get rid of that.